.changeset/kind-chairs-cough.md

```diff
@@ -0,0 +1,6 @@
+---
+"@medusajs/utils": patch
+---
+
+feat(utils): Backport metadata management
```

Comment:
add `modules-sdk`

Reply:
I did and then reverted man, I ve only changed the utils package :D